thing-management.md

```diff
@@ -115,36 +101,46 @@ Described in Node Provisioning Information Type Registry 3.1.2.4.
 
 Can be one of the following:
 
-- 1: Z-Wave is supported
-- 2: Z-Wave Long Range is supported
+- 0: Z-Wave is supported
```

Comment:
0:Z-Wave is supported and Long range is not supported. Or too obvious ?

Reply:
It's not said that more values will be available in the future.
I cannot imagine of any now, but it's possible from the frame construction now.